pytesseract/pytesseract.py

```diff
@@ -556,6 +581,71 @@ def image_to_osd(
     }[output_type]()
 
 
+def group_result(dict, config=None):
+    data = {}
+    groupped = {}
+    k = 0
+    for i in range(len(dict['line_num'])):
+        txt = dict['text'][i]
+        conf = int(dict['conf'][i])
+        block_num = dict['block_num'][i]
+        line_num = dict['line_num'][i]
+        par_num = dict['par_num'][i]
+
+        if not (txt == '' or txt.isspace()):
+            tup = {'text': txt, 'conf': conf}
+            if block_num in data:
+                if par_num in data[block_num]:
+                    if line_num in data[block_num][par_num]:
+                        data[block_num][par_num][line_num].append(tup)
+                    else:
+                        data[block_num][par_num][line_num] = [tup]
+                else:
+                    data[block_num][par_num] = {}
+                    data[block_num][par_num][line_num] = [tup]
+            else:
+                data[block_num] = {}
+                data[block_num][par_num] = {}
+                data[block_num][par_num][line_num] = [tup]
+
+    for _, b in data.items():
+        for _, l in b.items():
```

Comment:
you can use `.values()` to get only values

Reply:
Indeed, I will make the required changes.